### ISSUE TITLE
Clicking the + Button when no Board is present leads to FC

### ISSUE THREAD

**Issue by watschu** [stefan-niedermann/nextcloud-deck#252]
**Describe the bug**
When first starting the app, the + button in the header bar invites to click. Doing so leads to a Null Pointer Exception

This happens at least in versions 0.0.15 and 0.0.16

**To Reproduce**
Steps to reproduce
1. Start app, log into server.
2. If present, deleta all existing boards
3. Press the + button in the header (create column button)
4. See error:

**Expected behavior**
The Button in the header should have the same behaviour as the Floating Button (if no board available, create board, else create column in board)

**Screenshots**
![Screenshot_20200102-143513_Nextcloud Deck](https://user-images.githubusercontent.com/17567473/71669535-41645100-2d6d-11ea-943a-b767dc79fd3e.jpg)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S9
 - Android-Version: 9

**Stacktrace**
```
Version: 0.0.16

java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Long it.niedermann.nextcloud.deck.model.full.FullBoard.getLocalId()' on a null object reference
	at it.niedermann.nextcloud.deck.persistence.sync.SyncManager.lambda$createStack$10$SyncManager(SyncManager.java:395)
	at it.niedermann.nextcloud.deck.persistence.sync.-$$Lambda$SyncManager$W7n7FKfQ0Mqi70h1mudzbjNrGBg.run(Unknown Source:8)
	at java.lang.Thread.run(Thread.java:764)
```

**Comment by stefan-niedermann** [comment]
Totally true :) Good catch

**Comment by stefan-niedermann** [comment]
Fixed in `0.0.17`.
